src/sentry/search/eap/spans/formulas.py

```diff
@@ -18,6 +18,15 @@
 from sentry.search.eap.constants import RESPONSE_CODE_MAP
 from sentry.search.eap.utils import literal_validator
 
+TOTAL_SPAN_COUNT = Column(
```

Comment:
Could you add a comment explaining this is so we can reuse this def

Reply:
done!